src/components/WorkOrder/AddImage.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react';
+import { Camera } from './Image/Camera';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCamera, faTrash, faUpload } from '@fortawesome/free-solid-svg-icons';
+import api from '../../queries/api';
+import Header from '../Shared/Header';
+import dataURLtoBlob from 'blueimp-canvas-to-blob';
+import {
+  videoParentStyles,
+  videoStyles,
+  pictureButtonStyles,
+} from '../../styles/AddImage.css';
+
+class AddImage extends Component {
+  constructor() {
+    super();
+    this.camera = null;
+    this.state = {
+      capturedImage: null,
+      captured: null,
+      uploading: false,
+      hideVideo: false,
+    };
+  }
+
+  componentDidMount() {
+    // initialize the camera
+    this.canvasElement = document.createElement('canvas');
+    this.webcam = new Camera(
+      document.getElementById('webcam'),
+      this.canvasElement
+    );
+    this.webcam.setup().catch(() => {
+      alert('Error getting access to your camera');
+    });
+  }
+
+  captureImage = async () => {
+    const capturedData = this.webcam.takeBase64Photo({
+      type: 'jpeg',
+      quality: 0.8,
```

Comment:
Wondering how/if we'd want to tweak this setting...

Reply:
Leaving this as is for now. I agree that quality could be lowered to make the app less demanding of bandwidth (high quality jpegs could get large). My initial search about the quality parameter didn't shed much light on exactly how it works.